plottr/plot/mpl/plotting.py

```diff
@@ -206,3 +218,29 @@ def plotImage(ax: Axes, x: np.ndarray, y: np.ndarray,
     im = ax.imshow(z.T, aspect='auto', origin='lower',
                    extent=extent, **kw)
     return im
+    
+def plot_as_linecuts(ax: Axes, x: np.ndarray, y: np.ndarray,
+                     z: np.ndarray, cmap, **kw) -> Tuple[str,ScalarMappable]:
```

Comment:
Here cmap and **kw are not typed. 
cmap should be of type `matplotlib.colors.Colormap` e.g. here `colors.Colormap` should be correct.

**kw can just be typed as Any

Reply:
the other failures are not related to your pr

Reply:
They should be fixed by https://github.com/toolsforexperiments/plottr/pull/207